wagtail/wagtailadmin/menu.py

```diff
@@ -96,17 +115,30 @@ def is_shown(self, request):
         # show the submenu if one or more of its children is shown
         return bool(self.menu.menu_items_for_request(request))
 
+    def is_active(self, request):
+        return bool(self.menu.menu_items_active(request))
+
     def render_html(self, request):
-        return format_html(
-            """<li class="menu-item menu-{0}">
-                <a href="#" class="submenu-trigger {1}"{2}>{3}</a>
-                <div class="nav-submenu">
-                    <h2 class="{1}">{3}</h2>
-                    <ul>{4}</ul>
-                </div>
-            </li>""",
-            self.name, self.classnames, self.attr_string, self.label, self.menu.render_html(request)
-        )
+
+        self.menu.menu_items_for_request(request)
+        is_active = request.path.startswith(self.url)
```

Comment:
Looks like ``self.url`` is always ``#`` in ``SubmenuMenuItem`` so this would always be ``False``?